guides/v3.1.0/ember-inspector/component-tree.md

```diff
@@ -0,0 +1,50 @@
+The Components tab displays a collapsible representation of the views and components that are currently being rendered.  Selecting a component from the tree will open it in the [Object Inspector](../object-inspector/). 
+
+<img src="/images/guides/ember-inspector/component-tree-intro.png" width="680">
+
+Components will be displayed with curly braces.  Views are grayed out and not selectable.  Use the [View Tree](../view-tree) to get more information on Views that are being rendered.
```

Comment:
This link is missing the trailing slash, it should be

```
[View Tree](../view-tree/) 
``` 

and then it should fix the Travis build 👍